docs/DEVGUIDE.md

```diff
@@ -146,7 +146,7 @@ list:
     kubectl get servicebrokers,serviceclasses,serviceinstances,servicebindings
```

Comment:
not entirely related, but I get an error at this step:
`the server doesn't have a resource type "servicebrokers"`